docs/tutorials/kubernetes/kyverno.md

```diff
@@ -50,7 +50,7 @@ spec:
 
 ### Apply the policy to your Kubernetes cluster
 
-Ensure that you have Kyverno already deployed and running on your cluster -- for instance throught he Kyverno Helm Chart.
+Ensure that you have Kyverno already deployed and running on your cluster -- for instance thought he Kyverno Helm Chart.
```

Comment:
I think it should be `through`

Reply:
Thanks I fixed it 👍 
https://github.com/aquasecurity/trivy/pull/3171/commits/f146861a79d4bc065af65e3c13d1101eb631dfad